debug/src/index.js

```diff
@@ -5,8 +5,4 @@ initDebug();
 
 export { resetPropWarnings } from './check-props';
 
-export {
-	getCurrentVNode,
-	getDisplayName,
-	getOwnerStack
-} from './component-stack';
+export { captureOwnerStack } from './component-stack';
```

Comment:
Unfortunately we did backport this in #4860, and I don't see anything that demands removal, so can we make this additive instead? If those APIs don't get used much then that's fine, `/debug` is pretty isolated & non-probelmatic.

Reply:
The removed APIs have been added back. Sorry for missing the backport.

Reply:
No problem, we backported in the off chance it might've been useful to y'all.

Thanks!